aten/src/ATen/DLConvertor.cpp

```diff
@@ -60,6 +60,13 @@ DLDataType getDLDataType(const Tensor& t) {
     case ScalarType::QUInt2x4:
       TORCH_CHECK(false, "QUInt/QInt types are not supported by dlpack");
       break;
+    case ScalarType::Bits1x8:
+    case ScalarType::Bits2x4:
+    case ScalarType::Bits4x2:
+    case ScalarType::Bits8:
+    case ScalarType::Bits16:
```

Comment:
Suggest also doing 32, 64; also 24 and 48 if you are feeling spicy. I saw the comment saying we should also have the irregular bits like 7-bit but idk how that actually is supposed to work